examples/lambda-functions/get-all-items.ts

```diff
@@ -0,0 +1,87 @@
+import { APIGatewayProxyEvent, APIGatewayProxyResult, Context } from 'aws-lambda';
+import { Metrics } from '@aws-lambda-powertools/metrics';
+import { Logger } from '@aws-lambda-powertools/logger';
+import { Tracer } from '@aws-lambda-powertools/tracer';
+import { DynamoDB } from 'aws-sdk';
+
+// Create the PowerTools clients
+const metrics = new Metrics();
+const logger = new Logger();
+const tracer = new Tracer();
+
+// Create DynamoDB DocumentClient and patch it for tracing
+const docClient = tracer.captureAWS(new DynamoDB.DocumentClient());
```

Comment:
This won't work, since we are patching a specific client we need to use `tracer.captureAWSClient()` instead.

```ts
const docClient = tracer.captureAWSClient(new DocumentClient());
```